src/main.cpp

```diff
@@ -48,6 +48,7 @@ int main(int argc, char* argv[]) {
 
   UCI::loop(argc, argv);
 
+  TT.resize(0); // release hash first to avoid segfault on waiting for search completion
```

Comment:
In the previous patch revision, we released the memory on UCI:loop(). However, this caused a segfault when "stockfish bench" was executed. Now we should have a bit more luck on getting a clean CI run.

Reply:
not sure I understand the comment. 'first' means 'before .. what exactly'  ?

Reply:
We release hash and threads. So hash first. I suppose the comment could also say:

  // free hash before threads to avoid segfault on waiting for search completion

Reply:
Ah, sure.